resources/js/Components/Notifications/NewSuggestion.vue

```diff
@@ -15,7 +15,7 @@ const emit = defineEmits(['markedAsdone'])
       {{ notification.data.message }}
     </div>
     <span class="flex text-xs"
-      > {{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }}<span class="ml-1 font-bold">{{ notification.data.room.name }}</span></span
+      > {{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }} <span class="ml-1 font-bold">{{ notification.data.room.name }}</span></span
```

Comment:
Par exemple ici.

`{{ __('Sent by ') }}{{ notification.data.user.name }} {{ __('on ') }} <span class="ml-1 font-bold">{{ notification.data.room.name }}`

devient

`{{ __('Sent by') }}&nbsp;{{ notification.data.user.name }}&nbsp;{{ __('on') }}&nbsp;<span class="font-bold">{{ notification.data.room.name }}`

`&nbsp;` : Non breaking space HTML Character Entity

Reply:
dac c'est noté merci!